lib/runtime-core/src/cache.rs

```diff
@@ -59,16 +58,16 @@ impl WasmHash {
     /// is, in fact, a wasm module.
     pub fn generate(wasm: &[u8]) -> Self {
         let mut first_part = [0u8; 32];
-        let mut second_part = [0u8; 32];
+        // We don't use the second part for now, since the blake3 hash is 32 byte array
+        let second_part = [0u8; 32];
```

Comment:
This should be deleted -- it's too bad it produces a smaller digest, but it's probably fine